docs/templates/function_parameter_datatable.html.j2

```diff
@@ -1,22 +1,47 @@
-<label for="fork_selector">Filter by Fork: </label>
-<select id="fork_selector">
-    <option value="all">All Forks</option>
-    {% set default_fork = target_fork if target_fork in cases | map(attribute='fork') | list else cases | map(attribute='fork') | list | last %}
-    {% for fork in cases | map(attribute='fork') | unique %}
-    <option value="{{ fork }}" {% if fork == default_fork %}selected{% endif %}>{{ fork }}</option>
-    {% endfor %}
-</select>
-
-<label for="fixture_selector">Filter by Fixture Type: </label>
-<select id="fixture_selector">
-    <option value="all">All Fixture Types</option>
-    {% set default_fixture_type = "STATE_TEST" if "STATE_TEST" in cases | map(attribute='fixture_type') | list else cases[0].fixture_type %}
-    {% for fixture_type in cases | map(attribute='fixture_type') | unique %}
-    <option value="{{ fixture_type }}" {% if fixture_type == default_fixture_type %}selected{% endif %}>{{ fixture_type }}</option>
-    {% endfor %}
-</select>
-
-<table id="test_table" class="display nowrap">
+<div class="control_panel">
+    <div class="panel_row filters">
+    <div class="filter_wrapper">
+        <label for="fork_selector">
+         <span class="twemoji">
+            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
```

Comment:
Here's how this renders on the standalone page (Firefox):
![image](https://github.com/user-attachments/assets/1b1fe021-da32-4fae-9d3e-c7efb138c827)